learn-responsive-web-design-by-building-a-piano/index05.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Responsive Piano</title>
+</head>
+
+<body>
+
+    <div id="piano">
+        <div class="keys">
+            <div class="key"></div>
+            <div class="key"></div>
+            <div class="key"></div>
+            <div class="key"></div>
+            <div class="key"></div>
+            <div class="key"></div>
+            <div class="key"></div>
+        </div>
+    </div>
+ 
+</body>
+</html>
+
+<!-- 
+    Now you need to include an extra class value to style the
```

Comment:
The explanation here is fine, but I would recommend breaking this up into at least two lessons. Have the student add `black--key` to one of the divs first, then instruct them to add that value to the other divs.